dashboard/app/reporting.go

```diff
@@ -733,7 +733,7 @@ func loadOpenBugs(c context.Context) ([]*Bug, []*db.Key, error) {
 }
 
 func foreachBug(c context.Context, filter func(*db.Query) *db.Query, fn func(bug *Bug, key *db.Key) error) error {
-	const batchSize = 1000
+	const batchSize = 2000
```

Comment:
https://cloud.google.com/datastore/docs/concepts/limits

It says `Maximum number of keys allowed for a Lookup operation in the Datastore API`. Is it something different?

Reply:
I'll check performance diff. But it works 100%.